Resources/translations/validators.fr.xlf

```diff
@@ -4,7 +4,7 @@
         <body>
             <trans-unit id="1">
                 <source>This value is not a valid captcha.</source>
-                <target>Cette valeur n'est pas un captcha valide.</target>
+                <target>Vous devez cocher la case "Je ne suis pas un robot"</target>
```

Comment:
Since some users may be using a reCaptcha v1 key, where the checkbox is not present and a classic deformed text appears, I am not sure this proposal is a good idea.